README.md

```diff
@@ -1,5 +1,7 @@
 # Development project analysis pipeline
-## V_1.2
+## V_1.3
+
+### Refer to the associated [publication](https://www.nature.com/articles/s41467-020-18286-y) for additional details: Ellis, C. T., Skalaban, L. J., Yates, T. S., Bejjanki, V. R., Córdova, N. I., & Turk-Browne, N. B. (2020). Re-imagining fMRI for awake behaving infants. Nature Communications, 11(1), 1-12.
 
```

Comment:
no where here does it mention iBEAT -- may be nice to direct to that option in step 20?